www/js/tripconfirm/post-trip-prompt.js

```diff
@@ -118,8 +118,8 @@ angular.module('emission.tripconfirm.posttrip.prompt', ['emission.plugin.logger'
 
   var displayCompletedTrip = function(notification, state, data) {
     $rootScope.displayingIncident = true;
+    $rootScope.notificationData = notification.data;
       Logger.log("About to display completed trip from Notification");
-      $state.go("root.main.tripconfirm", notification.data);
```

Comment:
don't you need `$state.go` somewhere in here? Just setting the `notificationData` is not enough.

Reply:
I've got `$state.go` in `diary/list.js` instead.

Reply:
but that assumes that you are in the diary screen. Don't you need to `$state.go` to the diary first? What is your default start tab/state?

Reply:
In particular, look at the diffs for `incident/post-trip-prompt.js` that you are trying to adapt. We have a `$state.go` which you have removed

Reply:
Done.